ruby_connectors/lib/ruby_connectors/services/redis/workflow_store_service.rb

```
@@ -0,0 +1,59 @@
+        def get_workflow_data(workflow_key)
+          key = workflow_key(workflow_key)
+          data = @redis.hgetall(key)
+          return nil if data.empty?
+
+          {
+            status: data["status"],
+            result: data["result"] ? JSON.parse(data["result"]) : nil,
+            created_at: Time.zone.at(data["created_at"].to_i),
+            updated_at: data["updated_at"] ? Time.zone.at(data["updated_at"].to_i) : nil
+          }.with_indifferent_access
+        end
```

rubocop (1.68.0)

[convention] Assignment Branch Condition size for get_workflow_data is too high. [<2, 17, 3> 17.38/17]

(Metrics/AbcSize)